_drafts/2024-05-30-lessons-learned-from-the-latest-aviation-project-announcement.markdown

```diff
@@ -0,0 +1,14 @@
+---
+title: "Lessons Learned from the Latest Aviation Project"
```

Comment:
This is too vague for me. Who will remember/know which was the "Latest Aviation Project" in a couple of months?

Reply:
Lessons Learned from the Development of IS for Safety and Reliability of Aircrafts

Reply:
For example, or (System instead of IS) "Lessons Learned from the Development of a System for Safety and Reliability of Aircrafts"